src/frontend/src/binder/expr/function.rs

```diff
@@ -728,6 +728,11 @@ impl Binder {
                         binder.auth_context.user_name.clone(),
                     ))
                 }))),
+                ("user", guard_by_len(0, raw(|binder, _inputs| {
```

Comment:
Shall we:
* Add comments about why `user` is not sharing code with `session_user`
* Support `current_user` and `current_role` as aliases

Reply:
Sure, I will add it later in Beijing.